insights/tests/parsers/test_iris.py

```diff
@@ -32,6 +32,251 @@
+M/UX=255^#^66^$C(8)^^^^^
+MAIL=132^#^11^$C(8)^^^^^
+P-DEC=132^#^66^$C(8)^^^^^
+PK-DEC=150^#^66^$C(8)^^^^^
+PK-QUME=150^#^66^$C(8)^^^^^
+
+[Devices]
+0=0^TRM^C-IRIS Terminal^^^^Principal device^
+2=2^SPL^PK-DEC^^^^Spool LA120^
+47=47^MT^M/UX^^("auv":0:2048)^^Magnetic tape^
+48=48^MT^M/UX^^("avl":0:2048)^^Magnetic tape^
+57=57^BT^M/UX^^("auv":0:2048)^^Magnetic tape^
+58=58^BT^M/UX^^("avl":0:2048)^^Magnetic tape^
+SPOOL=2^SPL^PK-DEC^^^^Spool LA120^
+TERM=0^TRM^C-IRIS Terminal^^^^Windows Console^
+|LAT|=0^TRM^C-VT220^^^^Principal device^
+|PRN|=|PRN|^OTH^P-DEC^^"W"^^Windows Printer^
+|TNT|=0^TRM^C-VT220^^^^Principal device^
+|TRM|=0^TRM^C-IRIS Terminal^^^^Windows Console^
+
+[MagTapes]
+47=/dev/st0
+48=/dev/st1
+49=/dev/st0l
+50=/dev/st1l
+51=/dev/st0a
+52=/dev/st1a
+53=/dev/st0m
+54=/dev/st1m
+55=/dev/nst0
+56=/dev/nst1
+57=/dev/st0
+58=/dev/st1
+59=/dev/st0l
+60=/dev/st1l
+61=/dev/st0a
+62=/dev/st1a
+
+[config]
+LibPath=
+MaxServerConn=1
+MaxServers=2
+Path=
+PythonPath=
+bbsiz=-1
+console=,
+errlog=500
+globals=0,0,600,0,0,0
+gmheap=0
+history=500
+ijcbuff=512
+ijcnum=16
+jrnbufs=64
+locksiz=0
+memlock=0
+netjob=1
+nlstab=50
+overview=Linux/x86-64~UNIX
+pijdir=
+routines=0
+targwijsz=0
+udevtabsiz=24576
+wijdir=
+zfheap=0,0
+
+[Miscellaneous]
+AsyncDisconnectErr=0
+AsynchError=1
+BreakMode=1
+CollectResourceStats=0
+DisconnectErr=0
+ZDateNull=0
+ZaMode=0
+
+[ECP]
+ClientReconnectDuration=1200
+ClientReconnectInterval=5
+ServerTroubleDuration=60
+
+[Cluster]
+CommIPAddress=
+JoinCluster=0
+
+[LicenseServers]
+LOCAL=127.0.0.1,4002
+
+[Monitor]
+SNMPEnabled=0
+
+[IO]
+File=^%X364
+MagTape=^%XMAG
+Other=^%X364
+Terminal=^%X364
+
+[SQL]
+ANSIPrecedence=1
+AdaptiveMode=1
+AllowRowIDUpdate=0
+AutoParallel=1
+DropDelete=1
+ECPSync=0
+ExtrinsicFunctions=0
+FastDistinct=1
+IdKey=1
+IdTrxFrom=~ `!@#$%^&*()_+-=[]\{}|;':",./<>?
+IdTrxTo=
+LockThreshold=1000
+LockTimeout=10
+TODATEDefaultFormat=DD MON YYYY
+TimePrecision=0
+
+[SqlSysDatatypes]
+BIGINT=%Library.BigInt
+BIGINT(%1)=%Library.BigInt
+BINARY=%Library.Binary(MAXLEN=1)
+BINARY VARYING=%Library.Binary(MAXLEN=1)
+CLOB=%Stream.GlobalCharacter
+DATE=%Library.Date
+DATETIME=%Library.DateTime
+DATETIME2=%Library.DateTime
+DEC=%Library.Numeric(MAXVAL=999999999999999,MINVAL=-999999999999999,SCALE=0)
+DEC(%1)=%Library.Numeric(MAXVAL=<|'$$maxval^%apiSQL(%1,0)'|>,MINVAL=<|'$$minval^%apiSQL(%1,0)'|>,SCALE=0)
+DEC(%1,%2)=%Library.Numeric(MAXVAL=<|'$$maxval^%apiSQL(%1,%2)'|>,MINVAL=<|'$$minval^%apiSQL(%1,%2)'|>,SCALE=%2)
+DECIMAL=%Library.Numeric(MAXVAL=999999999999999,MINVAL=-999999999999999,SCALE=0)
+DECIMAL(%1)=%Library.Numeric(MAXVAL=<|'$$maxval^%apiSQL(%1,0)'|>,MINVAL=<|'$$minval^%apiSQL(%1,0)'|>,SCALE=0)
+DECIMAL(%1,%2)=%Library.Numeric(MAXVAL=<|'$$maxval^%apiSQL(%1,%2)'|>,MINVAL=<|'$$minval^%apiSQL(%1,%2)'|>,SCALE=%2)
+DOUBLE=%Library.Double
+DOUBLE PRECISION=%Library.Double
+
+[Telnet]
+DNSLookup=ON
+Port=23
+
+[Conversions]
+LastConvertTime=2023-07-20 08:58:29
+""".strip()
```

Comment:
Since most of the content of this case is not used in the following test, better keep the required lines as less as we can.